app/controllers/auctions_controller.rb

```diff
@@ -32,6 +32,14 @@ def update
 
   private
 
+  def find_auction
+    if params[:token]
+      Auction.find_by(token: params[:token])
```

Comment:
sure -- this seems pretty simple but I am down with doing that for consistency. What do you think we should call it?